--- xfce4-panel-4.14.0/plugins/actions/actions.c.orig	2019-04-10 00:23:04.000000000 +0200
+++ xfce4-panel-4.14.0/plugins/actions/actions.c	2019-08-29 23:51:32.239964536 +0200
@@ -155,7 +155,7 @@
     N_("Are you sure you want to log out?"),
     N_("Logging out in %d seconds."),
     "system-log-out",
-    NULL
+    "xfsm-logout"
   },
   { ACTION_TYPE_LOGOUT_DIALOG,
     "logout",
@@ -163,7 +163,7 @@
     N_("Log _Out..."),
     NULL, NULL, /* already shows a dialog */
     "system-log-out",
-    NULL
+    "xfsm-logout"
   },
   { ACTION_TYPE_SWITCH_USER,
     "switch-user",
@@ -171,7 +171,7 @@
     N_("_Switch User"),
     NULL, NULL, /* not needed */
     "system-users",
-    NULL
+    "xfsm-switch"
   },
   { ACTION_TYPE_LOCK_SCREEN,
     "lock-screen",
@@ -179,7 +179,7 @@
     N_("Loc_k Screen"),
     NULL, NULL, /* not needed */
     "system-lock-screen",
-    NULL
+    "xfsm-lockscreen"
   },
   { ACTION_TYPE_HIBERNATE,
     "hibernate",
@@ -188,7 +188,7 @@
     N_("Do you want to suspend to disk?"),
     N_("Hibernating computer in %d seconds."),
     "system-hibernate",
-    NULL
+    "xfsm-hibernate"
   },
   { ACTION_TYPE_HYBRID_SLEEP,
     "hybrid-sleep",
@@ -206,7 +206,7 @@
     N_("Do you want to suspend to RAM?"),
     N_("Suspending computer in %d seconds."),
     "system-suspend",
-    NULL
+    "xfsm-suspend"
   },
   { ACTION_TYPE_RESTART,
     "restart",
@@ -215,7 +215,7 @@
     N_("Are you sure you want to restart?"),
     N_("Restarting computer in %d seconds."),
     "system-reboot",
-    NULL
+    "xfsm-reboot"
   },
   { ACTION_TYPE_SHUTDOWN,
     "shutdown",
@@ -224,7 +224,7 @@
     N_("Are you sure you want to shut down?"),
     N_("Turning off computer in %d seconds."),
     "system-shutdown",
-    NULL
+    "xfsm-shutdown"
   }
 };
 
