--- xfce4-panel-4.2.3/configure.ac.orig	2005-11-05 18:07:46.000000000 +0100
+++ xfce4-panel-4.2.3/configure.ac	2005-11-10 05:43:42.000000000 +0100
@@ -36,7 +36,7 @@
 dnl Check for i18n support
 BM_I18N([xfce4-panel], [af ar az be bg bn_IN ca cs da de el en_GB eo es
                         es_MX et eu fa fr fi gu he hi hu hy it ja ko ku
-                        lt mr ms nb_NO nl pa pl pt_BR pt_PT ro ru sv sk
+                        lt mr ms nb nl pa pl pt_BR pt ro ru sv sk
                         sl ta tr uk vi zh_CN zh_TW])
 
 dnl Check for X11 installed
