diff -urN xfce4-panel-4.3.99.2/configure.ac xfce4-panel-4.3.99.2.new/configure.ac
--- xfce4-panel-4.3.99.2/configure.ac	2006-11-05 12:12:18.000000000 +0100
+++ xfce4-panel-4.3.99.2.new/configure.ac	2006-11-06 15:44:38.000000000 +0100
@@ -64,7 +64,7 @@
 AC_CHECK_FUNCS([sigaction])
 
 dnl Check for i18n support
-XDT_I18N([ar az be bg bn_IN ca cs de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu hy it ja ka ko lt mr ms nb_NO nl pa pl pt_BR pt_PT ro ru sk sv ta tr uk vi zh_CN zh_TW])
+XDT_I18N([ar az be bg bn_IN ca cs de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu hy it ja ka ko lt mr ms nb nl pa pl pt_BR pt ro ru sk sv ta tr uk vi zh_CN zh_TW])
 
 dnl Check for X11 installed
 XDT_CHECK_LIBX11_REQUIRE
