--- xfce4-panel-4.0.6/configure.ac.orig	2004-07-08 21:12:44.000000000 +0200
+++ xfce4-panel-4.0.6/configure.ac	2004-07-12 18:24:18.922696256 +0200
@@ -32,8 +32,8 @@
 AC_CHECK_FUNCS([sigaction])
 
 dnl Check for i18n support
-BM_I18N([xfce4-panel], [ar az bg ca cs de es es_MX et eu fa_IR fr fi hi hu it lt ms nl
-	 ja ko no pl pt_BR pt_PT ru sv ta tr uk vi zh_CN zh_TW])
+BM_I18N([xfce4-panel], [ar az bg ca cs de es es_MX et eu fa fr fi hi hu it lt ms nl
+	 ja ko nb pl pt_BR pt ru sv ta tr uk vi zh_CN zh_TW])
 
 dnl Check for X11 installed
 BM_LIBX11_REQUIRE
