--- xfce4-panel-4.3.90.1/configure.ac~	2006-06-29 23:18:32.000000000 +0200
+++ xfce4-panel-4.3.90.1/configure.ac	2006-06-29 23:19:24.000000000 +0200
@@ -47,9 +47,7 @@
 AC_CHECK_FUNCS([sigaction])
 
 dnl Check for i18n support
-XDT_I18N([ar az be bg bn_IN ca cs de el en_GB eo es es_MX et eu fa fi fr
-	  gu he hi hu hy it ja ko lt mr ms nb nl pa pl pt_BR pt ro
-	  ru sk sv ta tr uk vi zh_CN zh_TW ])
+XDT_I18N([ar az be bg bn_IN ca cs de el en_GB eo es es_MX et eu fa fi fr gu he hi hu hy it ja ko lt mr ms nb nl pa pl pt_BR pt ro ru sk sv ta tr uk vi zh_CN zh_TW ])
 
 dnl Check for X11 installed
 XDT_CHECK_LIBX11_REQUIRE
