--- ./po/LINGUAS.org	2008-10-28 01:03:00.879067182 +0100
+++ ./po/LINGUAS	2008-10-28 01:03:11.846149422 +0100
@@ -1,3 +1,3 @@
 # set of available languages (in alphabetic order)
-am ar az be bg bn_IN ca cs da de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu hy id it ja ka ko lt lv mk mr ms nb_NO nl pa pl pt_BR pt_PT ro ru si sk sv ta tr uk ur vi zh_CN zh_TW
+am ar az be bg bn_IN ca cs da de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu hy id it ja ka ko lt lv mk mr ms nb nl pa pl pt_BR pt ro ru si sk sv ta tr uk ur vi zh_CN zh_TW
 
--- ./configure.ac.org	2008-10-28 01:02:50.825734565 +0100
+++ ./configure.ac	2008-10-28 01:03:42.916146378 +0100
@@ -65,7 +65,7 @@ AC_CHECK_FUNCS([sigaction])
 AC_FUNC_MMAP()
 
 dnl Check for i18n support
-XDT_I18N([am ar az be bg bn_IN ca cs da de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu hy id it ja ka ko lt lv mk mr ms nb_NO nl pa pl pt_BR pt_PT ro ru si sk sv ta tr uk ur vi zh_CN zh_TW])
+XDT_I18N([am ar az be bg bn_IN ca cs da de dz el en_GB eo es es_MX et eu fa fi fr gl gu he hi hu hy id it ja ka ko lt lv mk mr ms nb nl pa pl pt_BR pt ro ru si sk sv ta tr uk ur vi zh_CN zh_TW])
 
 dnl Check for X11 installed
 XDT_CHECK_LIBX11_REQUIRE
