--- xfce4-panel-4.7.4/plugins/applicationsmenu/applicationsmenu.c~	2010-11-03 20:41:40.000000000 +0100
+++ xfce4-panel-4.7.4/plugins/applicationsmenu/applicationsmenu.c	2010-11-15 13:37:28.712464684 +0100
@@ -945,15 +945,16 @@
         }
       else
         {
-          /* lookup the xfce-applications.menu file */
-          filename = xfce_resource_lookup (XFCE_RESOURCE_CONFIG, DEFAULT_MENU);
-          if (G_LIKELY (filename != NULL))
-            menu = garcon_menu_new_for_path (filename);
-          g_free (filename);
+          /* load default menu */
+          menu = garcon_menu_new_applications ();
 
-          /* fallback to the default menu */
-          if (G_UNLIKELY (menu == NULL))
-            menu = garcon_menu_new_applications ();
+          /* fallback to the xfce-applications.menu file */
+          if (G_UNLIKELY (menu == NULL)) {
+            filename = xfce_resource_lookup (XFCE_RESOURCE_CONFIG, DEFAULT_MENU);
+            if (G_LIKELY (filename != NULL))
+              menu = garcon_menu_new_for_path (filename);
+            g_free (filename);
+	  }
         }
 
       if (menu != NULL
